src/libraries/System.Text.Json/tests/JsonNullTests.cs

```diff
@@ -46,16 +46,16 @@ public static void TestEquals()
             Assert.False(jsonNull1.Equals(new Exception()));
 
             // Null comparisons behave this way because of implicit conversion from null to JsonNull:
```

Comment:
Do we have an implicit conversion from `null` to `JsonNull`? How?

Looks like we should remove this comment. @joperezr, @bartonjs - do you know what this comment is trying to say?

Reply:
JsonNode defines many implicit conversions, the one from string can produce a JsonNull.

```C#
string s = null;
JsonNode sNode = s;
// sNode is JsonNull.Instance
```

If the calling code was

```C#
JsonNode someNode = ...;
string s = null;

if (someNode == s)
{
    ...
}
```

But since we don't define `operator==` or IEquatable on JsonNode (only the sealed node types), I don't see where ambiguity comes in.

Maybe this is left over from an earlier, more flexible model.  But I agree it should be deleted.

Reply:
> JsonNode defines many implicit conversions, the one from string can produce a JsonNull.

Yes, of course. Thanks!

> But I agree it should be deleted.

Great.